test/solidity-ibc/ICS20TransferTest.t.sol

```diff
@@ -257,6 +259,39 @@ contract ICS20TransferTest is Test, DeployPermit2, PermitSignature {
         assertEq(sequence, 42);
     }
 
+    function test_success_onAcknowledgementPacket() public {
```

Comment:
Nit: the name of this and the other test function you added might be good to just call something like `test_success_onAcknowledgemetnPacketCallback`?

I do realize this also tests onAcknowledgementPacket, but if you're looking for the test that verifies callbacks, this would be a little hard to find.